### PR TITLE
Fix spdx license and add repository field

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "contributors": [
     "John Gruen <john.gruen@gmail.com>"
   ],
+  "private": true,
```
Comment (pdehaan) [inline package.json]: Make sure no accidental pushes to npm.

File: package.json [hunk 2]
```diff
@@ -24,15 +25,16 @@
     "sign": "npm run clean:extension && npm run build:extension && web-ext sign --source-dir=build/extension && mv web-ext-artifacts/*.xpi ./addon.xpi",
     "run": "npm run build:extension && web-ext run --source-dir=build/extension",
     "lint": "eslint --color src",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npm run lint",
     "deploy": "gh-pages -d build/web",
     "release:base": "npm run clean && npm run build:web && npm run sign && mv addon.xpi build/web && npm run deploy",
```
Comment (pdehaan) [inline package.json]: Wasn't sure about this... I was tempted to try using `npm-run-all clean build:web sign` to be a bit less verbose. 🤷‍♀️

File: package.json [hunk 3]
```diff
@@ -24,15 +25,16 @@
...
     "release:dev": "ADDON_URL='https://lmorchard.github.io/ThemesRFun/addon.xpi' SITE_URL='https://lmorchard.github.io/ThemesRFun/' npm run release:base"
   },
   "engines": {
     "node": ">=6.2.0"
   },
-  "license": "MPL2",
+  "license": "MPL-2.0",
+  "repository": "lmorchard/ThemesRFun",
```
Comment (pdehaan) [inline package.json]: These two tweaks were to fix these warnings during `npm install`:

```
npm WARN themesrfun@0.0.9 No repository field.
npm WARN themesrfun@0.0.9 license should be a valid SPDX license expression
```

**Comment by pdehaan** [comment]
@lmorchard As part of our "full-service QA", can you confirm that this is legit:

```json
"author": "Les Orchard <ma@lmorchard.com>",
```

Wasn't sure if that should be "ma@" or "me@" (which it is on your GitHub profile page). But maybe you have a catch-all on lmorchard.com domain and it's all moot.

**Comment by lmorchard** [comment]
I do have a catch all on lmorchard.com. but that is also a typo 😞

**Pull request merged**